front/src/modules/ui/table/options/components/TableOptionsDropdown.tsx

```diff
@@ -18,13 +19,23 @@ export const TableOptionsDropdown = ({
 }: TableOptionsDropdownProps) => {
   const resetViewEditMode = useResetRecoilState(viewEditModeState);
 
+  const [, setIsDraggingAndSelecting] = useRecoilState(
+    isDraggingAndSelectingState,
+  );
+
+  const handleClose = () => setIsDraggingAndSelecting(true);
+
+  const handleOpen = () => setIsDraggingAndSelecting(false);
```

Comment:
we should actually implement it differently but we already have a ticket for that: https://github.com/twentyhq/twenty/issues/1669